### ISSUE TITLE
[Accessibility] Focus after closing settings

### ISSUE THREAD

**Issue by emrahcom** [jitsi/jitsi-meet#17366]
### What happened?

Our company had an accessibility scan done for Jitsi. This is the description of one problem in the accessibility report:

> When leaving the Settings menu, or any of its submenus, the focus should be set on the settings button. Currently it resets the focus order.

### Platform

- [x] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Any browser

### Relevant log output

```shell

```

### Reproducibility

- [x] The problem is reproducible on meet.jit.si

### More details?

<img width="693" height="609" alt="Image" src="https://github.com/user-attachments/assets/5ec81bd4-c834-4518-ac1e-6648a4ebe680" />
